dateparser_data/supplementary_language_data/date_translation_data/ru.yaml

```diff
@@ -56,7 +59,34 @@ relative-type:
         - послезавтра
 
 simplifications:
-    - ^час: 1 час
-    - ^минуту: 1 минуту
-    - ^секунду: 1 секунду
```

Comment:
Is this right that 3 cases are removed?

Reply:
I added this cases as regular expressions below.

Reply:
Aha I see, thanks! I checked that they still work 👍